packages/component-library/src/components/plmg-button/plmg-button.scss

```diff
@@ -363,26 +372,94 @@
     }
   }
 
-  // Control: Size
+  // Control: Size & icons
   &.small {
     padding: tokens.$plmg-spacing-x0-5 tokens.$plmg-spacing-x1;
     font-size: tokens.$plmg-font-size-x0-875;
     line-height: tokens.$plmg-font-lineheight-loose;
+
+    &.icon-button {
+      padding: tokens.$plmg-spacing-x1;
+    }
+
+    .icon-left {
+      margin-right: tokens.$plmg-spacing-x0-5;
+    }
+    .icon-right {
+      margin-left: tokens.$plmg-spacing-x0-5;
+    }
+    .icon-center {
+      font-size: centerIconFontSize(
+        tokens.$plmg-font-size-x0-875,
+        tokens.$plmg-font-lineheight-loose
```

Comment:
I'm not sure what this token is doing but the icon center small button is the incorrect size.
It is 55x55px rather than 37x37px.
It is in part something to do with `padding: tokens.$plmg-spacing-x1;` which is giving an icon padding of 16px rather than 8px.